index.html

```diff
@@ -124,6 +124,22 @@ <h2>Доп. Информация о расположении и контакты
       <a href="https://htmlacademy.ru">HTML Academy</a>
     </p>
   </footer>
+  <form class="#" action="#" method="post">
```

Comment:
```suggestion
  <form action="#" method="post">
```
Если ты не знаешь какой класс дашь форме - не пиши его

Reply:
Не исправил этот комментарий